Makefile

```diff
@@ -190,4 +189,4 @@ kubebuilder_2.3.1_${TEST_OS}_${TEST_ARCH}: kubebuilder_2.3.1_${TEST_OS}_${TEST_A
 envtest: kubebuilder_2.3.1_${TEST_OS}_${TEST_ARCH}
 	# You can add klog flags for debugging, like: -klog.v=10 -klog.logtostderr
 	# klogv2 flags just wraps our existing logrus.
-	go test -v ./internal/test/vk_envtest -envtest=true
+	KUBEBUILDER_ASSETS=$(PWD)/kubebuilder_2.3.1_${TEST_OS}_${TEST_ARCH}/bin go test -v ./internal/test/vk_envtest -envtest=true
```

Comment:
Does this work with

```Makefile
envtest: KUBEBUILDER_ASSETS := $(PWD)/kubebuilder_2.3.1_${TEST_OS}_${TEST_ARCH}/bin
envtest: kubebuilder_2.3.1_${TEST_OS}_${TEST_ARCH}
....
```

Just a nit since you the Makefile has merge conflicts anyhow...

Reply:
That doesn't. Also, part of the benefit of the change is to get the command line printed out that you need to run the tests "by hand".

Please review / merge this first:
https://github.com/virtual-kubelet/virtual-kubelet/pull/914

and I will rebase on top of master after that.